src/util/GameLogicUtils.cpp

```diff
@@ -114,4 +114,28 @@ namespace spy::util {
         // return random number >= change of failure
         return prob(gen) >= (1 - chance);
     }
+
+    bool GameLogicUtils::probabilityTestWithCharacter(spy::gameplay::State &s, character::Character &character,
```

Comment:
```suggestion
    bool GameLogicUtils::probabilityTestWithCharacter(const spy::gameplay::State &s, const character::Character &character,
```

Fix für Sonar Codesmell (zusammen mit Anmerkung unten)